packages/flame/test/components/mixins/has_paint_test.dart

```diff
@@ -37,68 +170,189 @@ void main() {
     );
 
     test(
-      'getPaint throws exception when used on genericless component',
+      'setPaint sets a paint',
       () {
-        final comp = _MyComponent();
+        final comp = _MyComponentWithType();
```

Comment:
Seems like it was like this before you started this PR, but all `comp` variables should be named `component`.

Reply:
It was before my PR but good point - I've added a commit that does this.